### DISCUSSION TITLE
[gitlabci-include] version prefix for renovate-runner is suddenly removed

### DISCUSSION THREAD

**Discussion by JuJup** [renovatebot/renovate#26104] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.81.0

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I'm using Renovate in Gitlab for my projects and it also updates itself on a regular basis. That worked for months now and suddenly it lost the `v` Prefix for template version v16.58.1. This leads to an error in the pipeline, as it can't find the reference to the correct template version anymore. The MR looked like this:

| Package | Type | Update | Change |
|---|---|---|---|
| [renovate-bot/renovate-runner](https://gitlab.com/renovate-bot/renovate-runner) | repository | patch | `v16.58.0` -> `16.58.1` |

It's a bit weird, as I didn't change anything in the Renovate config and it worked flawlessly before. Also, I couldn't find a breaking change in the release logs. Any idea, what might have happened here?

I made a [reproduction repo](https://github.com/JuJup/renovate-gitlab-template-version), but somehow it's working there.. I keep investigating and leave this here in case someone has the same struggles.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by JuJup** [comment]
Ok, now with template version `v16.58.4` it worked again in my repo and it updates successfully to the just released version `v16.59.0`.

**Comment by viceice** [comment, marked as answer]
it was a bug in renovate v37.79.0 which was fixed in v37.81.1

- https://github.com/renovatebot/renovate/issues/26059
- https://github.com/renovatebot/renovate/releases/tag/37.79.0
- https://github.com/renovatebot/renovate/releases/tag/37.81.1
